fedora38-ansible-latest/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM fedora:38
+LABEL maintainer="Sebastian Gumprich"
```

Comment:
I just copied this, but let me know if you don't want your name on it :sweat_smile:

Reply:
Feel free to add your name there. :)